parser_test.go

```diff
@@ -19,6 +19,40 @@ func TestFindAndJoinStrings(t *testing.T) {
 	assert.True(t, "asd,Asd" == res || "Asd,asd" == res)
 }
 
+func TestFillGeoAddress(t *testing.T) {
+	var text string
+	var re *regexp.Regexp
+	var registrant Registrant
+	text = "Address: Troya, Rim, JustAStreet"
```

Comment:
"Rim"?) maybe Rome?

Reply:
Any reference to real places is purely coincidental, all names are invented